packages/playground/scripts/build-env.sh

```diff
@@ -6,8 +6,7 @@ STELLAR_NETWORK="${STELLAR_NETWORK:=test}"
 TIMEOUT="${TIMEOUT:=10000}"
 PAGE_SIZE="${PAGE_SIZE:=20}"
 MINTING_URL="https://alpha.minting.tfchain.grid.tf"
-STATS_URL="https://stats.grid.tf"
-
+MANUAL_URL="https://www.manual.grid.tf"
```

Comment:
this will always overwrite the value in the environment.
```suggestion
MANUAL_URL="${MANUAL_URL:-https://www.manual.grid.tf}"
```

Reply:
There's no value in the environment. Only one value for all nets.

Reply:
okay, but this value can't be changed. If the manual link gets changed and exported, it won't reflect in the script.